### PR TITLE
Made a model to todo item, edited timezone

### LINKED ISSUE #5: Create model for TODO item
You should create a model for TODO item.

You need to define all fields (task title, due date etc) except remainders. Remainders will be added in the next version.

### PR DESCRIPTION
**Pull request by dandel10n**
Fixed #5

### CODE REVIEW

**Comment by m1kola** [comment]
In my opinion this PR is ready to merge.

File: todo/models.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+from django.db import models
+from django.utils import timezone
+
+
+class Task(models.Model):
+    #author = models.ForeignKey('auth.User')
+    title = models.TextField()
```
Comment (m1kola) [inline todo/models.py]: Ignore my [last comment](https://github.com/dandel10n/todo-list/pull/14#issuecomment-223588017).

We don't need `title` to be `TextField`. I think we can limit `title` with 255 symbols.

**Comment by dandel10n** [comment]
I fixed it

**Comment by m1kola** [comment]
It seems to me that we can't mark task as complete at the moment.

**Comment by dandel10n** [comment]
I've added is_completed field to the model

**Pull request merged**